force-app/main/default/core-functionality/flows/Thread_Set_To_Redaction.flow-meta.xml

```diff
@@ -1,5 +1,39 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Flow xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <actionCalls>
+        <name>Show_Toast_Message</name>
+        <label>Show Toast Message</label>
+        <locationX>314</locationX>
+        <locationY>758</locationY>
+        <actionName>c:fcFlowToaster</actionName>
+        <actionType>component</actionType>
+        <flowTransactionModel>CurrentTransaction</flowTransactionModel>
+        <inputParameters>
+            <name>duration</name>
+            <value>
+                <stringValue>10</stringValue>
+            </value>
+        </inputParameters>
+        <inputParameters>
+            <name>message</name>
+            <value>
+                <stringValue>Samtaletråden ble satt til sladding.</stringValue>
```

Comment:
```suggestion
                <stringValue>Henvendelsen ble satt til sladding.</stringValue>
```

Reply:
Takk for tilbakemelding, jeg kan godt legge til en Failed toast også :)

Reply:
Jeg tror ikke det skal være behov for å gjøre det nå, men gjerne gjør det hvis du vil 😄